src/utils/Utils.ts

```diff
@@ -1709,4 +1710,32 @@ export default class Utils {
     }
     return moment.duration(totalDurationSecs, 's').format('h[h]mm', { trim: false });
   }
+
+  public static handleExpetionDetailedMessages(exception: AppError): void {
```

Comment:
I'm surprised that this method is only used by logActionAppException?
What about the others like 'logActionBackendException'?

Reply:
Done